core/src/main/java/hudson/FilePath.java

```diff
@@ -563,39 +584,64 @@ public Void invoke(File dir, VirtualChannel channel) throws IOException, Interru
      * @param compression
      *      Compression mode of this tar file.
      * @since 1.292
+     *
+     * @see FilePath#untar(FilePath, TarCompression, Consumer)
      * @see #untarFrom(InputStream, TarCompression)
      */
     public void untar(final FilePath target, final TarCompression compression) throws IOException, InterruptedException {
+        untar(target, compression, (Consumer<List<String>> & Serializable)(a) -> {});
```

Comment:
does this not produce warnings at runtime about the serialization of this Lambda?

Reply:
without that Serializable it fails, and had to be added to remote calls (FilePathSecureTest)

Reply:
I was more thinking that it would produce a warning like anonymous classes do. (I have not ever tried to send a Lambda over remoting)  https://www.jenkins.io/doc/developer/extensibility/serialization-of-anonymous-classes/

Reply:
without the serialised interface you get this error https://ci.jenkins.io/blue/organizations/jenkins/Core%2Fjenkins/detail/PR-5061/5/tests